pkg/cloud/aws/services/kubeadm/scheme.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeadm
```

Comment:
can we rename this package to avoid conflict with kubeadm packages?
`kubeadm-codecs`, maybe?

Reply:
we should only be importing the kubeadm types and the convention there to rename would be `kubeadmv1beta1`